proof/courier.go

```diff
@@ -615,12 +614,12 @@ func (b *BackoffHandler) initialDelay(ctx context.Context,
 		return nil
 	}
 
-	locatorHash, err := proofLocator.Hash()
+	locatorStr, err := proofLocator.LogString()
```

Comment:
Non-blocking, but you can make a prefix logger out of this. Then all the logs can start with the locator prefix with some nice formatting.

Reply:
Thanks, I'll look into that. I think there’s room to standardise our log messages a bit more across the codebase.